strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/validation/groupid/LowercaseValidator.java

```diff
@@ -0,0 +1,29 @@
+package org.carlspring.strongbox.storage.validation.groupid;
```

Comment:
As this is shared between the group id and the artifact id, it would be best to move this one level higher up -- in the `org.carlspring.strongbox.storage.validation`.

Reply:
working on it